src/app/edit-maprule/preset/preset.html

```diff
@@ -0,0 +1,168 @@
+<div class="preset-container" [formGroup]="editMapRule.form">
+  <div formArrayName="presets">
+    <ng-container *ngFor="let preset of editMapRule.form.controls.presets.controls; let presetIndex=index">
+      <div class="preset-card" [formGroupName]="presetIndex">
+        <div class="preset-card-header preset-card-panel-header-hidden">
+          <input
+            placeholder="{{ 'Preset ' + (presetIndex + 1) }}"
+            class="form-input form-input-preset"
+            formControlName="presetName" />
+          <div>
+            <button class="preset-button" id="show-preset-button" (click)="showPresetPanel(presetIndex)">
+              <i class="fa fa-plus-square-o"></i>
+            </button>
+            <button class="preset-button" id="remove-preset-button" (click)="removePreset(presetIndex)">
+              <i class="fa fa-trash-o"></i>
+            </button>
+          </div>
+        </div>
+        <div #preset class="preset-card-panel preset-card-panel-hidden">
```

Comment:
I don't think I mentioned, so this seems like a good place...

Both disabled and presets start with the hidden class applied, then once they are initted, the form data is checked and a decision is made to remove them (to show them). 

Effect is...

1. if an existing maprule is loaded, presets are all closed, and (if disabled has data) it is open
2. If new maprule, both are open.